src/thx/Tuple.hx

```diff
@@ -74,7 +72,11 @@ Provides a string representation of the Tuple
 **/
 @:forward(_0, _1)
 abstract Tuple2<T0, T1>({ _0 : T0, _1 : T1 }) from { _0 : T0, _1 : T1 } to { _0 : T0, _1 : T1 } {
-  inline public static function of<A, B>(_0: A, _1: B): Tuple2<A, B> 
+/**
+Constructs an instance of `Tuple2` the 2 required value. This is required
+because Tuple2.new.bind(...) crashes the compiler.
+**/
+  inline public static function of<A, B>(_0: A, _1: B): Tuple2<A, B>
```

Comment:
I usually call this methods `create`. Many people also use `from`. That said I like `of` so I am confused.